crates/common/src/contracts.rs

```diff
@@ -265,3 +268,12 @@ mod tests {
         let _decoded = abi::decode(&params, args).unwrap();
     }
 }
+
+/// Helper function to convert CompactContractBytecode ~> ContractBytecodeSome
+pub fn compact_to_contract(contract: CompactContractBytecode) -> ContractBytecodeSome {
+    ContractBytecodeSome {
+        abi: contract.abi.unwrap(),
+        bytecode: contract.bytecode.unwrap().into(),
+        deployed_bytecode: contract.deployed_bytecode.unwrap().into(),
+    }
```

Comment:
Let's not unwrap here—rather return a Result

Reply:
yep, def cleaner, thanks!